CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/providers/TransactionTypeNodeProvider.java

```diff
@@ -125,9 +126,13 @@ public void newActiveGraph(final Graph graph) {
                 SchemaConceptUtilities.getChildConcepts(schemaFactory.getRegisteredConcepts())
                         .forEach(childConcept -> concepts.add(childConcept.getClass()));
                 schemaLabel.setText(String.format("%s - %s", schemaFactory.getLabel(), GraphNode.getGraphNode(graph).getDisplayName()));
-                transactionTypes.addAll(SchemaTransactionTypeUtilities.getTypes(concepts));
-                Collections.sort(transactionTypes, (final SchemaTransactionType a, final SchemaTransactionType b) -> a.getName().compareToIgnoreCase(b.getName()));
-
+                transactionTypes.addAll(
```

Comment:
This is not what we want to do as this will prevent two different types with the same name appearing, despite that being valid. Instead try replacing...
```
final Set<Class<? extends SchemaConcept>> concepts = new HashSet<>();
concepts.addAll(schemaFactory.getRegisteredConcepts());
SchemaConceptUtilities.getChildConcepts(schemaFactory.getRegisteredConcepts())
    .forEach(childConcept -> concepts.add(childConcept.getClass()));
```
with...
```
final Set<Class<? extends SchemaConcept>> concepts = schemaFactory.getRegisteredConcepts();
```
...as I think the issue is to do with child concepts being added twice (as they are also calculated in ```SchemaVertexTransactionUtilities```).